src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -11,6 +11,7 @@ import {
 import {colors} from '../../../utils/colors';
 import {fonts} from '../../../utils/fonts';
 import MemoDomainProfilePicture from '../../../assets/icon/DomainProfilePictureEmptyState';
+import TopicsProfilePictureEmptyState from '../../../assets/icon/TopicsProfilePictureEmptyState';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `TopicsProfilePictureEmptyState` is added but it's not clear if this module exists and is correctly exported from the specified path. Please ensure that the module is available and properly exported.
```diff
+ import TopicsProfilePictureEmptyState from '../../../assets/icon/TopicsProfilePictureEmptyState';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->